examples/http-failover/http_failover.bal

```diff
@@ -3,58 +3,55 @@ import ballerina/log;
 import ballerina/runtime;
 
 // Create an endpoint with port 8080 for the mock backend services.
-endpoint http:Listener backendEP {
-    port: 8080
-};
+listener http:Listener backendEP = new(8080);
 
 // Define the failover client end point to call the backend services.
-endpoint http:FailoverClient foBackendEP {
-    timeoutMillis: 5000,
-    failoverCodes: [501, 502, 503],
-    intervalMillis: 5000,
-    // Define set of HTTP Clients that needs to be Failover.
-    targets: [
-        { url: "http://localhost:3000/mock1" },
-        { url: "http://localhost:8080/echo" },
-        { url: "http://localhost:8080/mock" }
-    ]
+http:FailoverClient foBackendEP = new({
+        timeoutMillis: 5000,
+        failoverCodes: [501, 502, 503],
+        intervalMillis: 5000,
+        // Define set of HTTP Clients that needs to be Failover.
```

Comment:
Shall we change this to "Define a set of HTTP Clients that are targeted for failover."